### PR TITLE
Expand new contributors message in comments form

### LINKED ISSUE #6096: Expanded "be friendly to newcomers" message in comment forms
Stack Overflow has this nice message we could adapt from in our own comment form!

image

Would anyone like to give this a try?

**Changes to be made:** https://github.com/publiclab/plots2/blob/master/app/views/notes/_comments.html.erb#L18 

Check if a user is a new contributor using user.is_new_contributor method, and show comment placeholder accordingly. 

****

This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://code.publiclab.org/#r=all), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @publiclab/reviewers to get it labelled. Or, if this is not your first time, try to solve it yourself!

### PR DESCRIPTION
**Pull request by Tlazypanda**
Fixes #6096

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

# Screenshots  
Screenshot from 2020-01-16 11-53-22

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@jywarren have changed the comment placeholder for new contributors as mentioned although I feel we should also add that hover feature over new-contributor badge.Or in this also maybe we can have a separate div for this message instead of making changes in the comment placeholder.Let me know your thoughts and if I should create an issue for the same.Thanks:)

File: config/locales/en.yml [hunk 1]
```diff
@@ -207,6 +207,9 @@ en:
       post_placeholder: "Help the author refine their post, or point them at other
         helpful information on the site. Mention users by @username to notify them
         of this thread by email"
+      post_placeholder_new_contributor: "%{author} is a new contributor.Take care in
+      asking for clarification,commenting and answering.Be nice and check out our
+      Code of Conduct."
```
Comment (seabl) [inline config/locales/en.yml]: Could you edit the message to insert spaces after commas and periods?

Comment (Tlazypanda) [reply config/locales/en.yml]: @seabl updated the pr according to the changes requested.

**Comment by Tlazypanda** [comment]
@seabl @jywarren Ready for a review :) Thanks :v:

**Comment by seabl** [comment]
Awesome! I think this is ready.

**Comment by jywarren** [comment]
I like it too! The idea of making it in a div would let us link to the code of conduct, though, do you think it's worth doing for that reason? Or shall we do it in a follow-up? 

Thanks a lot, this will be great!!!

**Comment by Tlazypanda** [comment]
@jywarren have updated this pr itself with the div...updated the screenshot in the pr description.Could not find the wave hands icon in font awesome so used peace :v: instead.This is the initial draft kindly let me know what other changes can be made to make it better :smile: and the badge hover thing should I open a separate issue for that? Thanks :)

**Comment by jywarren** [comment]
This is so cool!

image

**Comment by jywarren** [comment]
Great work!!! 🎉 

We really appreciate it!!!

**Comment by jywarren** [comment]
I wonder if we should move the welcoming text into a "feature" so that site admins can edit it? What do you think?

https://github.com/publiclab/plots2/blob/master/doc/DATA_MODEL.md#features

It could be a nice follow-up?

**Comment by Tlazypanda** [comment]
@jywarren Sure, that sounds great :v: Will start by creating an issue and working on it right away! I am kinda new to this feature thing so might take some time :sweat_smile: Thanks :tada:

**Comment by jywarren** [comment]
no rush, and thanks for the help!

**Pull request merged**